src/pyOpenMS/pxds/IDFilter.pxd

```diff
@@ -19,82 +19,146 @@ cdef extern from "<OpenMS/FILTERING/ID/IDFilter.h>" namespace "OpenMS":
+        void updateHitRanks(libcpp_vector[ProteinIdentification]& identifications) nogil except + # wrap-doc:Updates the hit ranks on all peptide or protein IDs
 
-        void removeUnreferencedProteins(libcpp_vector[ProteinIdentification]& proteins, libcpp_vector[PeptideIdentification]& peptides) nogil except +
+        void removeUnreferencedProteins(libcpp_vector[ProteinIdentification]& proteins, libcpp_vector[PeptideIdentification]& peptides) nogil except + # wrap-doc:Removes protein hits from the protein IDs in a 'cmap' that are not referenced by a peptide in the features or if requested in the unassigned peptide list
 
-        void updateProteinReferences(libcpp_vector[PeptideIdentification]& peptides, libcpp_vector[ProteinIdentification]& proteins, bool remove_peptides_without_reference) nogil except +
+        void updateProteinReferences(libcpp_vector[PeptideIdentification]& peptides, libcpp_vector[ProteinIdentification]& proteins, bool remove_peptides_without_reference) nogil except + # wrap-doc:Removes references to missing proteins. Only PeptideEvidence entries that reference protein hits in 'proteins' are kept in the peptide hits
 
         bool updateProteinGroups(libcpp_vector[ProteinGroup]& groups, libcpp_vector[ProteinHit]& hits) nogil except +
+            # wrap-doc:
+                #   Update protein groups after protein hits were filtered
+                #   -----
+                #   :param groups: Input/output protein groups
+                #   :param hits: Available protein hits (all others are removed from the groups)
+                #   :return: Returns whether the groups are still valid (which is the case if only whole groups, if any, were removed).
 
-        void removeEmptyIdentifications(libcpp_vector[PeptideIdentification]& ids) nogil except +
-        void removeEmptyIdentifications(libcpp_vector[ProteinIdentification]& ids) nogil except +
+        void removeEmptyIdentifications(libcpp_vector[PeptideIdentification]& ids) nogil except + # wrap-doc:Removes peptide or protein identifications that have no hits in them
+        void removeEmptyIdentifications(libcpp_vector[ProteinIdentification]& ids) nogil except + # wrap-doc:Removes peptide or protein identifications that have no hits in them
 
-        void filterHitsByScore(libcpp_vector[PeptideIdentification]& ids, double threshold_score) nogil except +
-        void filterHitsByScore(libcpp_vector[ProteinIdentification]& ids, double threshold_score) nogil except +
+        void filterHitsByScore(libcpp_vector[PeptideIdentification]& ids, double threshold_score) nogil except + # wrap-doc:Filters peptide or protein identifications according to the score of the hits. Only peptide/protein hits with a score at least as good as 'threshold_score' are kept. Score orientation (are higher scores better?) is taken into account
```

Comment:
`Score orientation (are higher scores better?) is taken into account`
This should probably be removed from this and the statement below - is more an internal comment.
@jpfeuffer Do you know if it is taken into account?

Reply:
Fixed. Thank you for pointing it out @oliveralka .

Reply:
Yes. One has to set higherscorebetter in each PeptideIdentification for this to work correctly.
I think this note is actually quite important.

Reply:
Since it is very unintuitive.

Reply:
Yes, that really is unintuitive and should be in the comment, but probably in a different fashion than before, which seems more like a question to me than a statement.

Reply:
@rahul799 Would you please add a short statement, then it should be finished.

Reply:
@oliveralka can you please tell me the short statement that can be written. I am facing problems while writing as I have less context.